source/ui/x11/X11EventLoop.ooc

```diff
@@ -4,6 +4,7 @@ import include/x11
 import ../[EventLoop, DisplayWindow, GuiEvent]
 import UnixWindow
 
+version(!android && (unix || apple)) {
```

Comment:
I have both this and the other version in this PR and it seems to compile for android.